docs/connecting-your-data/assignment-tables/randomization-sdk.md

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Create assignment table with Eppo Randomization SDK
+
+# Overview
+
+If you don't currently have a third party feature flagging solution integrated, you can use Eppo’s SDK to randomly assign subjects to your Eppo experiment variations. 
```

Comment:
Could we use the abbreviated Overview in this doc: https://www.notion.so/eppo/DRAFT-SDK-Developer-Documentation-7969949576a0432ab1db43887f0dd623